behavior/datagen/gen.py

```diff
@@ -185,9 +185,8 @@ def _benchbase_init(self, benchmark, benchbase_results_dir):
         """
         try:
             # Copy the BenchBase config to the output directory.
-            input_cfg_path = self.dir_benchbase_config / f"{benchmark}_config.xml"
-            shutil.copy(input_cfg_path, benchbase_results_dir)
-            cfg_path = (benchbase_results_dir / f"{benchmark}_config.xml").absolute()
+            cfg_path = (self.dir_benchbase_config / f"sample_{benchmark}_config.xml").absolute()
```

Comment:
With this PR, how would we use a custom config file in `datagen`? I'm open to suggestions, but maybe it can retain the old functionality, i.e., try to find a config from the old path. Only if one doesn't exist, then use the BenchBase default one instead.

Reply:
This functionality is why they were originally there.  Now I just edit files within `artifacts` instead of `config`, and we have fewer files in this repository.  What this PR would remove is the ability to persist customized configuration files.  Still, we can't programmatically configure BenchBase, which is probably what we really want.  I'm not sure if we want this change.  It makes the repository cleaner but doesn't bring us closer to programmatic configuration or provide and new features.  The main reason I submitted it was the original change request in #12